doc/gitian-building-bu.md

```diff
@@ -0,0 +1,34 @@
+Gitian build
+============
+
+This is guide take for granted that you are using Ubuntu bionic 18.04 as host OS. the aim of the document is to be able to produce deterministic binaries using gitian-tools and docker containers.
+
+
+Prerequisite
+-------------
+
+These are steps that as to be executed once and that don't need to be repeated for every new gitian build process.
+
+```bash
+sudo apt install git apt-cacher-ng ruby docker.io
+sudo usermod -a -G docker $USER
+cd ~/src
```

Comment:
```gnome-session-quit```  # log out and back in so group change takes effect

Reply:
Actually issuing this command:

```bash
exec su -l $USER
```
from the same terminal that will be used to run the `bin/gbuild` should do trick.

Reply:
fixed